casper/src/main/scala/coop/rchain/casper/util/rholang/RegistrySigGen.scala

```diff
@@ -28,19 +28,19 @@ final case class InsertSigned(pk: Hex, value: (Long, Contract), sig: Hex) {
   def contract = value._2
 
   override def toString() = s"""
-    |new
-    |  ${contract.varName}, rs(`rho:registry:insertSigned:ed25519`), uriOut
-    |in {
-    |  contract ${contract.varName}(...) = {
-    |     ...
-    |  } |
-    |  rs!(
-    |    \"${pk}\".hexToBytes(),
-    |    (${nonce}, bundle+{*${contract.varName}}),
-    |    \"${sig}\".hexToBytes(),
-    |    *uriOut
-    |  )
-    |}
+     |new
```

Comment:
why indent 3 space rather than 2?

Reply:
I don't know how this happened. Will fix.